services/pkg/buildbot/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM  ghcr.io/void-linux/void-glibc-full:20240526R1
+
+RUN xbps-install -Suy xbps && xbps-install -uy python3 tini git curl
+# version must be synced with ../buildbot-worker
+RUN python3 -m venv /venv && \
+	/venv/bin/pip3 install wheel && \
+	/venv/bin/pip3 install 'buildbot[tls,bundle]~=4.0.0' treq \
+		buildbot-prometheus buildbot-netauth \
+		'git+https://github.com/classabbyamp/buildbot-void-theme.git'
```

Comment:
do we want to bring `buildbot-void-theme` under the void github namespace? I don't intend for it to ever be released on pypi as it's really just for us, we can grab it from git.

Reply:
Yes, this should come under the void namespace.